justfile

```diff
@@ -51,9 +51,3 @@ count-warnings:
 
 find-warnings:
 	@cargo check 2>&1 | grep -A1 -e 'warning' | grep --only-matching -e '-->[^:]*' | sort | uniq -c | sort -nr
-
-@update-todo:
-	./etc/update-todo.sh
-
-@count-failures:
-	./etc/count-tests.sh
```

Comment:
This doesn't save meta data to the repository. It's meant to be used during development as ephemeral data on the terminal.